### ISSUE TITLE
IPP - configure app in login.gov sandbox env 

### ISSUE THREAD

**Issue by djmunoz1** [department-of-veterans-affairs/va.gov-team#66635]
As a developer I need to connect the IPP app to the login.gov sandbox environment so that I can start testing the integration.

ACCEPTANCE CRITERIA:
- refer to instructions on https://developers.login.gov/
- Create an account in the Login.gov sandbox dashboard https://idp.int.identitysandbox.gov/
- Select between OpenID Connect (OIDC) or SAML protocol implementation protocols and understand which user attributes are required
- IPP application can read User Locator data from Login.gov (UUID, email, encryption key)

DEF OF DONE:
- [x] IPP Proofing Agent Screens are able to access User Locator data

**Comment by mateoclarke** [comment]
We'll first attempt this using the [NextAuth library](https://next-auth.js.org/) and with a [test Login.gov sandbox app](https://dashboard.int.identitysandbox.gov/service_providers/4002) configured with OIDC.

Here's [the branch](https://github.com/department-of-veterans-affairs/va-in-person-identity-proofing/pull/8) I'm kicking off.

In this, we'll need to create a custom provider for Login.gov which makes this a bit more complicated than a more common SSO setup: https://next-auth.js.org/configuration/providers/oauth#using-a-custom-provider

**Comment by rdvyas** [comment]
During team meeting we determined that we are close to completing this integration. There are some outstanding technical questions that we need Login.gov teams input on. We have a meeting scheduled on Jan 2 to triage these issues. Also we will be raising this issue during out Login.gov weekly sync to determination point of contact on Login.gov for technical issues.

**Comment by lmorris3** [comment]
@rdvyas what is "User Locator data"? Is it (UUID, email, encryption key)?

**Comment by rdvyas** [comment]
During 2/14 demo we were able to show integration of our app with IdP solution. We can close this ticket as we have established a connection between out app and the Login.gov solution using OIDC protocol. 

We were able to demonstrate pulling Login.gov user data for both Veteran and Internal User using PIV. This has been an integrated and we can close this ticket as completed. cc @mateoclarke @moanu404
